
project hardware/qcom/display-caf/msm8996/
diff --git a/hardware/qcom/display-caf/msm8996/sdm/libs/core/display_hdmi.cpp b/hardware/qcom/display-caf/msm8996/sdm/libs/core/display_hdmi.cpp
index c01f71b1..63289b3c 100644
--- a/hardware/qcom/display-caf/msm8996/sdm/libs/core/display_hdmi.cpp
+++ b/hardware/qcom/display-caf/msm8996/sdm/libs/core/display_hdmi.cpp
@@ -163,7 +163,6 @@ DisplayError DisplayHDMI::OnMinHdcpEncryptionLevelChange(uint32_t min_enc_level)
 uint32_t DisplayHDMI::GetBestConfig(HWS3DMode s3d_mode) {
   uint32_t best_index = 0, index;
   uint32_t num_modes = 0;
-  HWDisplayAttributes best_attrib;
 
   hw_intf_->GetNumDisplayAttributes(&num_modes);
 
